### DISCUSSION TITLE
TrinityCore as a fully open-source WotLK blizzlike server

### DISCUSSION THREAD

**Discussion by CraftedRO** [TrinityCore/TrinityCore#27180] [Suggestions]
Well since someone is paying I guess for https://tcubuntu.northeurope.cloudapp.azure.com to be up , 
I was thinking why don't you make it an open source WotLK blizzlike server as it is now but bringing more people by advertise even here.
My opinion is that doing this will involve more people by playing here to test and report occured issues and why not to fix them properly.
I don't wanna give ChromieCraft as example , but something like that.
Im sure that there are a lot of people interested in this and will be a smart move.
Probably you will claim that this is against copyright , but meh the entire emulator world is somehow against and they still exist.

**Comment by jackpoz** [comment]
> Well since someone is paying I guess for https://tcubuntu.northeurope.cloudapp.azure.com to be up

That would be my personal server using Azure Credits from Visual Studio MPN (Microsoft Partner Network) https://docs.microsoft.com/en-us/visualstudio/subscriptions/program-mpn from the company I work for, giving me 150$ a month I can spend on Azure resources. On October it used 93$, leaving plenty for other work-related activites.

Consider it as a "temporary" TC server that could disappear any moment (like if I change job or if my monthly credits run out). It's a good place to test issues on clean TC without any custom change and it runs with Address Sanitizer https://github.com/google/sanitizers/wiki/AddressSanitizer , allowing to spot crashes and memory issues even just by playing.

I am not so interested into having 100 players online, 5 would be already many IMO 😃
